### ISSUE TITLE
Unable to generate correct diagram with 'socialization' gem

### ISSUE THREAD

**Issue by mizurnix** [voormedia/rails-erd#215]
In my project I am using [socialization](https://github.com/cmer/socialization) gem. It kind of messes with the DB directly so probably it's not so unexpected that diagram was not generated properly and had errors specifically with the tables created by this gem.

Still, I thought I'd share this problem in order to let someone more competent see where or not some more generic issue is hiding behind this.

This is the type of error I receive when trying to run 'rake erd':

- Warning: Ignoring invalid association :follower on Socialization::ActiveRecordStores::Follow (polymorphic interface Follower does not exist)
- Warning: Ignoring invalid association :followable on Socialization::ActiveRecordStores::Follow (polymorphic interface Followable does not exist)

**Comment by kerrizor** [comment]
Thanks for reporting this! It'll probably not be something we get to any time soon (unless someone really gets a 🐝  in their 👒 ) but there's been a few other issues with AR relationships and how we approach them, so it is good to have this reported. I'll try to schedule some time to look into it as part of a 2.0.

**Comment by nroose** [comment]
Yeah, still polymorphic relationships are not supported. For us this is a significant number of important model associations.

**Comment by kerrizor** [comment]
Closing as stale (5+ years old with no recent activity). If you're still experiencing this issue, please open a new issue with current version details. Thanks! 🙏
